### PR TITLE
[Clover step-10] Sort the task list in order of creation date and time

### PR DESCRIPTION
**Pull request by clover-zhao**
### Summary/概要
**Description**:

Implemented a task sorting function based on the `created_at` timestamp both incresing and decresing order This provides users with an organized view of their tasks.

**Changes Made**:
1. Updated the `index` action in the `TasksController` to use the new sorting function when fetching tasks for display.
2. Modified the `index` view of tasks to correctly display tasks in the sorted order.
3. Created a button on the tasks page that triggers the sorting function, giving users the option to sort tasks by creation time.
4. Wrote tests for the new sorting function to ensure that tasks are sorted correctly.
6. Updated relevant views to display the 'created at' date for each task

### How to check the behavior/動作確認
- Sort by newest
  Screenshot 2023-08-02 at 10 19 45

- Sort by oldest
  
  Screenshot 2023-08-02 at 10 19 51

- Test result
  Screenshot 2023-08-02 at 10 21 34

### CODE REVIEW

**Comment by wataru0** [comment]
The title is alamin!

**Comment by clover-zhao** [comment]
> The title is alamin!

Corrected it! Thank you very much.

File: myapp/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -27,10 +27,8 @@ def render_not_found
   # Custom method to render the 500 Internal Server Error page
   def render_internal_server_error(exception)
     # Log the exception for debugging purposes
-    byebug
     Rails.logger.error "500 Internal Server Error: #{exception.message}"
     render template: 'errors/500.html', status: :internal_server_error
   end
```
Comment (anneju) [inline myapp/app/controllers/application_controller.rb]: pls remove the empty line

File: myapp/app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -1,6 +1,11 @@
 class TasksController < ApplicationController
   def index
     @tasks = Task.all
```
Comment (anneju) [inline myapp/app/controllers/tasks_controller.rb:3]: maybe not relevant to the task this time, but does the training ask you to do pagination in upcoming tasks? Usually we don't pull all records.

**Review by anneju — COMMENTED** [review]
some question~ (maybe irrelevant to this task 😅)

File: myapp/app/controllers/application_controller.rb (hunk 1 above)
Comment (clover-zhao) [reply myapp/app/controllers/application_controller.rb]: Sure, empty line removed.

File: myapp/app/controllers/tasks_controller.rb (hunk 2 above)
Comment (clover-zhao) [reply myapp/app/controllers/tasks_controller.rb:3]: Based on the training specifications, none of the task steps require pagination.

Comment (anneju) [reply myapp/app/controllers/tasks_controller.rb:3]: just double checked again, the pagination is at [step 14](https://github.com/Fablic/training/blob/develop/steps_en.md#step-14-lets-add-pagination). Let's add it later~

**Review by anneju — APPROVED** [review]
looks good~

**Pull request merged**